solutions.js

```diff
@@ -242,13 +242,13 @@ solutions.icyHot = function(temp1, temp2){
 // Java > Warm-up1 > loneTeen
 // We'll say that a number is "teen" if it is in the range 13..19 inclusive.
 // Given 2 int values, return true if one or the other is teen, but not both.
-solutions.loneTeen = function(a, b){
-	if (a >= 13 || a <= 19){
-		return true;
-		}
-	else{
-		return false;
-	}
+solutions.loneTeen = function(a, b) {
+	var isTeen = function(num) {
+		return num >= 13 && num <= 19;
+	};
+	var isTeenA = isTeen(a);
+	var isTeenB = isTeen(b);
+	return isTeenA && !isTeenB || !isTeenA && isTeenB;
```

Comment:
This could be rewritten as isTeenA != isTeenB but I'm not sure that is as clear.